webapp/app.py

```diff
@@ -128,19 +140,52 @@ def layout(self):
             self.make_symbol_table(),
 
             html.H1(children='Harlock Hardfork Table'),
-            self.make_nem_table()
+            self.make_nem_table(),
+
+            dcc.Interval(
+                id='load-interval',
+                interval=60000,  # in milliseconds
+                n_intervals=0
+            )
         ])
 
+        def update_graphs(n_intervals):
+            mtime = os.path.getmtime(self.resources_path / 'nem_harvesters.csv')
+            if mtime > self.mtime:
+                self.mtime = mtime
+                print(f'Updating at check {n_intervals}, modified time: {mtime}')
+                self.reload()
+                symbol_figure = self.make_symbol_figure()
+                symbol_allnodes_figure = self.make_symbol_allnodes_figure()
+                nem_figure = self.make_nem_figure()
+                symbol_table = self.make_symbol_table()
+                nem_table = self.make_nem_table()
+                return symbol_figure, symbol_allnodes_figure, nem_figure, symbol_table.data, nem_table.data  # pylint: disable=no-member
+            raise PreventUpdate()
```

Comment:
do you intend to raise this unconditionally?

Reply:
Yes. This is intentional. 

Before, it would always reload the files and redo the graphs even when there was no new data. Now, it only refreshes when new data. 

The way that Dash works, your only choices are Output something, or raise an error.